public/src/components/channelManagement/bannerDesigns/ColourInput.tsx

```diff
@@ -1,53 +1,64 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 
 import { TextField } from '@material-ui/core';
-import { hexColourStringRegex } from '../../../utils/bannerDesigns';
-import { Control, useController } from 'react-hook-form';
+import {
+  hexColourStringRegex,
+  hexColourToString,
+  stringToHexColour,
+} from '../../../utils/bannerDesigns';
+import { useForm } from 'react-hook-form';
 import { EMPTY_ERROR_HELPER_TEXT } from '../helpers/validation';
+import { HexColour } from '../../../models/bannerDesign';
 
 const colourValidation = {
   value: hexColourStringRegex,
   message: 'Colours must be a valid 6 character hex code e.g. FF0000',
 };
 
 interface Props {
-  control: Control;
+  colour: HexColour;
```

Comment:
it's not perfect, but I think it's nice to be able to work with the real model here